### DISCUSSION TITLE
Does the current release of nerdctl not support using Nydus Snapshotter ?

### DISCUSSION THREAD

**Discussion by zhaoshane** [containerd/nerdctl#1224] [Q&A] [answered]
Based on the doc for [using nydus snapshotter,](https://github.com/containerd/nerdctl/blob/master/docs/nydus.md), this feature requires nerdctl 0.22 or above, but in fact the latest revision found was 0.21.
![image](https://user-images.githubusercontent.com/44110085/178912926-70710c93-4c36-45ad-8af0-04b1f4d7a1e1.png)

Does it means nerdctl is not yet supporting this feature before new version is released.

**Comment by junnplus** [comment, marked as answer]
Yep, We haven't released 0.22 yet,  maybe the last few days.

**Reply by AkihiroSuda** [reply]
v0.22.0 is now available https://github.com/containerd/nerdctl/releases/tag/v0.22.0
